posthog/api/event_definition.py

```diff
@@ -15,9 +19,12 @@ class Meta:
             "query_usage_30_day",
         )
 
+    def update(self, event_definition: EventDefinition, validated_data):
+        raise PermissionDenied("This is an Enterprise plan feature.")
```

Comment:
Probably worth raising a specific error code for this as `403` is actually used to mean other things. How about creating an `EnterpriseFeatureException` (exceptions.py), could probably use `402` code?